test/format/config.c

```diff
@@ -285,6 +286,25 @@ config_table(TABLE *table, void *arg)
             config_single(table, "btree.value_min=20", false);
     }
 
+    /*
+     * Limit the rows to 1000000 if the realloc exact and realloc malloc configs are on and not all
+     * explicitly set. Realloc exact config allocates the exact amount of memory, which causes a new
+     * realloc call every time we append to an array. Realloc malloc turns a single realloc call to
+     * a malloc, a memcpy, and a free. The combination of both will significantly slow the
+     * execution.
+     */
+    if ((!config_explicit(NULL, "debug.realloc_exact") ||
+          !config_explicit(NULL, "debug.realloc_malloc") || !config_explicit(NULL, "runs.rows") ||
```

Comment:
With CONFIG.stress, runs.rows will be explicit but should also be a config_range in your new code.

Reply:
Yes, I consider rows still randomly generated in this case so we need to limit it.